src/style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__item {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.item {
+  margin: 0 0 4px;
+}
+
+.card__code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.code {
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__star {
+  display: flex;
+  flex-direction: row;
+}
+
+.star4__stars {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.star1 .star4__stars:nth-child(1),
+.star2 .star4__stars:nth-child(-n+2),
+.star3 .star4__stars:nth-child(-n+3),
+.star4 .star4__stars:nth-child(-n+4),
+.star5 .star4__stars:nth-child(n) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__rating_feedback {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price {
+  margin-bottom: 16px;

```

Comment:
the same thing here. you can simply move this property to `card__price` classname